model/salesemployee.ts

```diff
@@ -0,0 +1,8 @@
+export class SalesEmployee {
```

Comment:
you aren't instantiating this, so it could just be a type 

`export type SalesEmployee`